src/views/TabView/TabView.js

```diff
@@ -56,7 +54,18 @@ type Props = {
   },
 };
 
-class TabView extends PureComponent<void, Props, void> {
+let defaultInitialLayout = undefined;
+if (Platform.OS === 'android') {
+  // fix for https://github.com/react-native-community/react-native-tab-view/issues/312
+  const { width, height } = Dimensions.get('window');
+  defaultInitialLayout = { width, height };
+}
+
```

Comment:
could you please change this to something like

```
const windowDimensions = Dimensions.get('window')
const defaultInitialLayout = Platform.select({
  ios: undefined,
  android: { width: windowDimensions.width, height: windowDimensions.height },
  android: { width: 1, height: 0 } // or rather this because that's what they say here https://github.com/react-native-community/react-native-tab-view/issues/297
})
```